.github/ISSUE_TEMPLATE/update_entry.md

```diff
@@ -13,11 +12,11 @@ assignees: ''
 
 ## Data
 
-<!-- Check the fields that changed below and provide the new value -->
-
-- [ ] `name`: <!-- New name -->
-- [ ] `url`: <!-- New url -->
-- [ ] `difficulty`: <!-- New difficulty: easy/medium/hard/impossible -->
-- [ ] `notes`: <!-- Updated notes -->
-- [ ] `email`: <!-- New e-mail address -->
-- [ ] `domains`: <!--New e-mail subject -->
+- `name`: <!-- New Name -->
+- `url`: <!-- New Url -->
+- `difficulty`: <!-- New Difficulty: easy/medium/hard/impossible -->
+- `notes`: <!-- New Notes -->
+- `email`: <!-- New Email Address -->
+- `email_subject`: <!-- New Email Subject -->
+- `email_body`: <!-- New Email Body -->
+- `domains`: <!-- New Domains for out [Browser Extensions](https://github.com/jdm-contrib). -->
```

Comment:
Why removing the checkboxes?

Reply:
Most people seem to remove them, and besides, what's the use case even for them?

Reply:
Sounded easier to just check what changed, but I guess leaving it blank has the same meaning, so let's try that out instead then.

There are more complex issue templates now, but I don't have the time to play with them nor I think it will be a great improvement for us here.

Reply:
"Don't fix it if it ant broke"